.gitignore

```diff
@@ -6,12 +6,16 @@
 
 .idea/misc.xml
 
+.idea/preferred-vcs.xml
```

Comment:
I'll add the file then. I just confirmed that I have the latest version and I always use WebStorm's GitHub association while working on our project.